### PR TITLE
Add alpha wallet option and remove Fortmatic

### LINKED ISSUE #2050: Add Alpha wallet
Add WalletConnect based Alpha wallet using web3-react

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Fixes #2050

Adds alpha wallet option based on wallet-connect
Removes Fortmatic wallet option

Screenshot 2023-02-23 at 16 56 57

### CODE REVIEW

**Review by elena-zh — CHANGES_REQUESTED** [review]
Hey @nenadV91 , nice!

Some notes: 
1. Wallet icon is too tiny on the Connect wallet modal
logo

2. It would be great to show the wallet icon on waiting modals(currently. we show WC icon instead)
alph
3. (very impotrant): I can't place swap/limit orders being connected to this wallet:
order
image
4. Impossible to connect the wallet when Mainnet is not selected (enabled) there (however, it should be the wallet's issue)

**Comment by elena-zh** [comment]
As for mobile version, some similar notes I left in https://github.com/cowprotocol/cowswap/pull/2044#issuecomment-1441911875


OR can we navigate a user into wallet's integrated browser (like we do for mobile MM or Coinbase) when press on the Alpha button?

**Comment by nenadV91** [comment]
@elena-zh 
1 and 2 should be fixed now
3. I've also tried on gnosis chain. Approve and wrap/unwrap works but not swap/limit orders

**Comment by elena-zh** [comment]
Hey @nenadV91 , case 2 is still reproducible. 
Case 1 LGTM!

Some more details for the case 3: I can sign transactions when open CoW Swap inside their wallet in an integrated browser. So the issue is relevant for WC only.

**Comment by nenadV91** [comment]
@elena-zh I've disabled this wallet option in wallet modal also in integrated browsers.
For the case 2 this is what I see
Screenshot 2023-02-28 at 11 24 08

**Comment by elena-zh** [comment]
Hey @nenadV91 , I have cleared out my local storage and did hard reloads several times, but still I see this icon :( 
image

Anyways, I can't approve the PR until we resolve the issue with orders signing :(

**Comment by alfetopito** [comment]
> @elena-zh 1 and 2 should be fixed now 3. I've also tried on gnosis chain. Approve and wrap/unwrap works but not swap/limit orders

@nenadV91 Since this wallet definitely doesn't work, we should add to the list of unsupported wallets

https://github.com/cowprotocol/cowswap/blob/c2ed0796ff810ce060a7656021f4f3a9c1a6cbe3/src/custom/constants/index.ts#L53

**Comment by elena-zh** [comment]
> > @elena-zh 1 and 2 should be fixed now 3. I've also tried on gnosis chain. Approve and wrap/unwrap works but not swap/limit orders
> 
> @nenadV91 Since this wallet definitely doesn't work, we should add to the list of unsupported wallets
> 
> https://github.com/cowprotocol/cowswap/blob/c2ed0796ff810ce060a7656021f4f3a9c1a6cbe3/src/custom/constants/index.ts#L53

Hey @alfetopito , I should mention that it works when open CoW Swap app inside the wallet in their integrated browser. So it is partially working. Issue with signing orders is reproducible only when connected to it using WC

File: src/cow-react/modules/account/containers/AccountDetails/index.tsx [hunk 1]
```diff
@@ -57,6 +60,7 @@ import { isMobile } from 'utils/userAgent'
 import UnsupporthedNetworkMessage from 'components/UnsupportedNetworkMessage'
 import { SupportedChainId as ChainId } from 'constants/chains'
 import { useDisconnectWallet } from '@cow/modules/wallet/api/hooks/useDisconnectWallet'
+import { getIsAlphaWallet, getIsAmbireWallet, getIsZengoWallet } from '@src/custom/connection/utils'
```
Comment (shoom3301) [inline src/cow-react/modules/account/containers/AccountDetails/index.tsx:63]:
```suggestion
import { getIsAlphaWallet, getIsAmbireWallet, getIsZengoWallet } from 'connection/utils'
```

**Review by shoom3301 — APPROVED** [review]

**Comment by alfetopito** [comment]
> > > @elena-zh 1 and 2 should be fixed now 3. I've also tried on gnosis chain. Approve and wrap/unwrap works but not swap/limit orders
> > 
> > 
> > @nenadV91 Since this wallet definitely doesn't work, we should add to the list of unsupported wallets
> > https://github.com/cowprotocol/cowswap/blob/c2ed0796ff810ce060a7656021f4f3a9c1a6cbe3/src/custom/constants/index.ts#L53
> 
> Hey @alfetopito , I should mention that it works when open CoW Swap app inside the wallet in their integrated browser. So it is partially working. Issue with signing orders is reproducible only when connected to it using WC

I see!
In that case I would argue that it should be enabled only when the wallet is injected. Should be blocked when connected via WC IMO.

**Comment by nenadV91** [comment]
@alfetopito I would wait a bit with marking it as unsupported. We are trying to communicate the issue to the Alpha wallet team.

**Comment by nenadV91** [comment]
Closing this PR in favor of new one https://github.com/cowprotocol/cowswap/pull/2099

**Pull request closed without merging**